packages/components/CHANGELOG-FIGMA-FOUNDATIONS.md

```diff
@@ -1,4 +1,56 @@
-# [HDS Foundations UI Kit](https://www.figma.com/design/uX4OEaJQdWfzULADchjAeN/HDS-Foundations-v2.0?m=auto&node-id=2916-4&t=vnWfSubCHQU6J1U3-1)
+# [HDS Foundations UI Kit v2.0](https://www.figma.com/design/uX4OEaJQdWfzULADchjAeN/HDS-Foundations-v2.0?m=auto&node-id=2916-4&t=vnWfSubCHQU6J1U3-1)
+
+## December 20th, 2024
+
+Released HDS Foundations UI Kit v2.0.
+
+**Deprecated** HDS Foundations UI Kit v1.0. It will no longer receive updates or support. 
```

Comment:
[suggestion] consistency with the Components section.

```suggestion
- Deprecated HDS Foundations UI Kit v1.0. It will no longer receive updates or support. 
- Released HDS Foundations UI Kit v2.0.
```